frontend/pages/projects/_id/affective-annotation/index.vue

```diff
@@ -700,9 +735,18 @@ export default {
       }
     },
     async confirm() {
-      if(this.project.isSummaryMode || this.project.isOthersMode || this.project.isEmotionsMode) {
-        this.canConfirm = this.isAllAffectiveDataAdded()
-      }
+      if(this.project.isCombinationMode) {
+        // should be refactored
+        this.hasValidEntries.isSummaryMode = this.isAllAffectiveDataAdded()
+        this.hasValidEntries.isOthersMode = this.isAllAffectiveDataAdded()
+        this.hasValidEntries.isEmotionsMode = this.isAllAffectiveDataAdded()
```

Comment:
I think this comment can be removed now, since I have merged the fix. Sorry that I forgot to remove the comment before merging. 😅